src/graph/RadarTrackObjectsNode.cpp

```diff
@@ -160,7 +180,7 @@ void RadarTrackObjectsNode::enqueueExecImpl()
 
 std::vector<rgl_field_t> RadarTrackObjectsNode::getRequiredFieldList() const
 {
-	return {XYZ_VEC3_F32, DISTANCE_F32, AZIMUTH_F32, ELEVATION_F32, RADIAL_SPEED_F32};
+	return {XYZ_VEC3_F32, DISTANCE_F32, AZIMUTH_F32, ELEVATION_F32, RADIAL_SPEED_F32, ENTITY_ID_I32};
```

Comment:
[Docs](https://github.com/RobotecAI/RobotecGPULidar/blob/feature/q2-features/include/rgl/api/core.h#L858) should be updated.